src/authentication/web3-login/index.tsx

```diff
@@ -21,22 +21,25 @@ export class Web3Login extends React.Component<Web3LoginProperties, Web3LoginSta
   render() {
     const { error, isConnecting, onSelect } = this.props;
 
+    const errorText =
+      error === Web3LoginErrors.PROFILE_NOT_FOUND
+        ? 'The wallet you connected is not associated with a ZERO account'
+        : error;
+
     return (
-      <div className={c('')}>
-        {isConnecting ? (
-          <div className={c('connecting')}>
-            <Button isDisabled={true} endEnhancer={<IconLoading2 isFilled={true} className={c('spinner')} size={20} />}>
-              Waiting for wallet confirmation
-            </Button>
+      <div {...cn('')}>
+        <div {...cn('login')}>
+          <div {...cn('select-wallet')}>
+            <WalletSelect isConnecting={isConnecting} onSelect={onSelect} />
           </div>
-        ) : (
-          <div className={c('login')}>
-            <div className={c('select-wallet')}>
-              <WalletSelect isConnecting={false} onSelect={onSelect} />
+          {error && (
+            <div {...cn('error-container')}>
```

Comment:
Looks to me like the error styles for this flow differ to other alert error styles in the app i.e. font-size and line-height spacing. Other uses of Alert don't require the container for the padding here either. Will note to look at zUI and will ping design another question related to this.